integreat_cms/cms/views/region_condition/region_condition_actions.py

```diff
@@ -48,6 +48,11 @@ class RegionConditionResource(resources.ModelResource):
 
     num_outdated_pages = fields.Field(column_name=_("Number of outdated pages"))
 
+    has_translation_package_been_booked = fields.Field(
```

Comment:
```suggestion
    translation_budget = fields.Field(
```
I know "has_" as a prefix only for BooleanFields. Since this is not one anymore, I think that we should also change the variable name (if that's easily doable :) )